eclair-core/src/main/scala/fr/acinq/eclair/blockchain/WatcherTypes.scala

```diff
@@ -43,7 +43,7 @@ sealed trait Watch {
  *
  * @param replyTo         actor to notify once the transaction is confirmed.
  * @param txId            txid of the transaction to watch.
- * @param publicKeyScript when using electrum, we need to specify a public key script; any of the output scripts should work.
+ * @param publicKeyScript output script of any of the transaction's outputs.
```

Comment:
Should I completely remove these fields? They were only necessary for Electrum.

Reply:
I think yes.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1750/commits/447cba4187a56c7d974c73feb3e533d8f7369708